docs/openstack-floating-ips.md

```diff
@@ -1,6 +1,6 @@
 # Openstack Floating Ips
 
-To read more about Openstack Floating Ips using the [upstream docs](https://docs.openstack.org/python-openstackclient/pike/cli/command-objects/floating-ip.html).
+Read more about Openstack Floating Ips using the [upstream docs](https://docs.openstack.org/python-openstackclient/pike/cli/command-objects/floating-ip.html).
```

Comment:
lets put the latest docs link here.

https://docs.openstack.org/python-openstackclient/latest/cli/command-list.html

Reply:
updated